rbac/migration_tool/migrate.py

```diff
@@ -188,7 +189,7 @@ def migrate_data(exclude_apps: list = [], orgs: list = [], write_relationships:
         return
 
     count = 0
-    tenants = Tenant.objects.exclude(tenant_name="public")
```

Comment:
it looks that ready flag doesn't help here - it looks we need to improve query to exclude tenants which does not need to be replicated otherwise it will go over 13M tenants  with no roles, groups,.. and it will take hours.

Reply:
Answered in meeting, existing tenants will have set ready=True.